includes/admin/class-wcmypa-admin.php

```diff
@@ -463,65 +481,105 @@ public function showOrderActions($order): void
+        $addReturn    = WCMP_Export::EXPORT_RETURN;
+        $pluginUrl    = WCMYPA()->plugin_url();
+        $baseUrl      = 'admin-ajax.php?action=' . WCMP_Export::EXPORT;
+
+        return [
             $addShipments => [
-                "url" => admin_url("$baseUrl&request=$addShipments&order_ids=$order_id"),
-                "img" => WCMYPA()->plugin_url() . "/assets/img/export.svg",
-                "alt" => __("Export to MyParcel", "woocommerce-myparcel"),
+                'url' => admin_url("$baseUrl&request=$addShipments&order_ids=$orderId"),
+                'img' => "{$pluginUrl}/assets/img/export.svg",
+                'alt' => __('action_export_to_myparcel', 'woocommerce-myparcel'),
             ],
             $getLabels    => [
-                "url" => admin_url("$baseUrl&request=$getLabels&order_ids=$order_id"),
-                "img" => WCMYPA()->plugin_url() . "/assets/img/print.svg",
-                "alt" => __("Print MyParcel label", "woocommerce-myparcel"),
+                'url' => admin_url("$baseUrl&request=$getLabels&order_ids=$orderId"),
+                'img' => "{$pluginUrl}/assets/img/print.svg",
+                'alt' => __('action_print_myparcel_label', 'woocommerce-myparcel'),
             ],
             $addReturn    => [
-                "url" => admin_url("$baseUrl&request=$addReturn&order_ids=$order_id"),
-                "img" => WCMYPA()->plugin_url() . "/assets/img/return.svg",
-                "alt" => __("Email return label", "woocommerce-myparcel"),
+                'url' => admin_url("$baseUrl&request=$addReturn&order_ids=$orderId"),
+                'img' => "{$pluginUrl}/assets/img/return.svg",
+                'alt' => __('action_email_return_label', 'woocommerce-myparcel'),
             ],
         ];
+    }
 
-        $consignments = self::get_order_shipments($order);
+    /**
+     * @param array $listingActions
+     * @param array $metaPps
+     *
+     * @return array
+     */
+    public static function updateExportButtonForPps(array $listingActions, array $metaPps): array
+    {
+        $pluginUrl = WCMYPA()->plugin_url();
 
-        if (empty($consignments)) {
-            unset($listing_actions[$getLabels]);
-        }
+        $listingActions[WCMP_Export::EXPORT_ORDER]['img'] = "{$pluginUrl}/assets/img/myparcel.svg";
 
-        $processed_shipments = self::get_order_shipments($order);
-        if (empty($processed_shipments) || $shipping_country !== 'NL') {
-            unset($listing_actions[$addReturn]);
-        }
+        foreach ($metaPps as $metaPpsFeedback) {
 
-        $display    = WCMYPA()->setting_collection->getByName(WCMYPA_Settings::SETTING_DOWNLOAD_DISPLAY) === 'display';
-        $attributes = [];
+            if (is_array($metaPpsFeedback) && $metaPpsFeedback[self::META_PPS_EXPORTED]) {
 
-        if ($display) {
-            $attributes["target"] = "_blank";
-        }
+                $listingActions[WCMP_Export::EXPORT_ORDER]['alt'] = __(
```

Comment:
Waar is dit voor?

Reply:
Feedback naar de gebruiker, wanneer de order al eerder is ge"exporteerd krijgt de gebruiker dit als hint bij het knopje.